Networking/Networking/Model/Product/ProductDownload.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 
 /// Represents a ProductDownload entity.
 ///
-public struct ProductDownload: Codable, GeneratedFakeable {
+public struct ProductDownload: Codable, GeneratedFakeable, GeneratedCopiable {
```

Comment:
Q: Was that added just for consistency? I don't see any usage of `ProductDownload.copy()`

Reply:
You are right, I end up not needed it! Removed in https://github.com/woocommerce/woocommerce-ios/pull/3816/commits/58c30f4d688327084439fe6f2bdaf69adeb12ead

Thanks!